src/components/ContributeAs.js

```diff
@@ -119,8 +128,8 @@ const useForm = ({ onProfileChange }) => {
 /**
  * Search is displayed if 5 or more profiles are passed in.
  */
-const ContributeAs = ({ onProfileChange, personal, profiles, defaultSelectedProfile, ...fieldProps }) => {
-  const { getFieldError, getFieldProps, onFieldChange, onSearch, onChange, state } = useForm({ onProfileChange });
+const ContributeAs = ({ intl, onProfileChange, personal, profiles, defaultSelectedProfile, ...fieldProps }) => {
+  const { getFieldError, getFieldProps, onFieldChange, onSearch, onChange, state } = useForm({ intl, onProfileChange });
```

Comment:
Do we need `intl` in `useForm`? I don't see it used there

Reply:
Good catch. Removing it...